cli/commands.go

```diff
@@ -24,6 +24,10 @@ func setDatabase(opts *Options) func(ctx *cli.Context) error {
 	return func(ctx *cli.Context) error {
 		var db = ctx.String("db")
 
+		if opts.DB == nil {
+			opts.Logger.Fatal("DB can't be empty, if you wanna use it")
```

Comment:
want to use it? )

Reply:
@kpeo https://github.com/cryptopay-dev/yaga/pull/15#issuecomment-361253695